src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1258,6 +1271,102 @@ tr.status-warning td {
     margin: 0 1.5em 0 0;
 }
 
+#summary .terminal {
+    --header-status-request-method-color: var(--gray-400);
+    --header-metadata-key: var(--gray-400);
+    --header-metadata-value: var(--gray-300);
+
+    background: var(--terminal-bg);
+    border: solid var(--terminal-border-color);
+    border-width: 30px 4px 4px 4px;
+    border-radius: 3px 3px 0 0;
+    box-shadow: none;
+    color: var(--gray-100);
+    padding: 10px 15px;
+    position: relative;
+}
+#summary .terminal .status-request-method {
+    margin-right: 10px;
+}
+#summary .terminal.status-success .status-response-status-code {
+    margin-right: 1.5em;
+    text-transform: uppercase;
+}
+#summary .terminal.status-warning,
+#summary .terminal.status-warning .status-response-status-text {
+    color: var(--terminal-warning-color);
+}
+#summary .terminal.status-warning .status-response-status-code {
+    background: var(--terminal-warning-bg);
+    color: var(--black);
+    margin-right: 1.5em;
+    padding: 1px 5px;
+    text-transform: uppercase;
+}
+#summary .terminal.status-warning,
+#summary .terminal.status-warning .status-response-status-text {
+    color: var(--terminal-warning-color);
+}
+#summary .terminal.status-error,
+#summary .terminal.status-error .status-response-status-text {
+    color: var(--terminal-error-color);
+}
+#summary .terminal.status-error .status-response-status-code {
+    background: var(--terminal-error-bg);
+    color: var(--black);
+    margin-right: 1.5em;
+    padding: 1px 5px;
+    text-transform: uppercase;
+}
+#summary .terminal.status-error,
+#summary .terminal.status-error .status-response-status-text {
+    color: var(--terminal-error-color);
+}
+.macos #summary .terminal:before {
+    background-color: rgba(255, 255, 255, 0.3);
+    border-radius: 50%;
+    box-shadow: 0 0 0 2px rgba(255, 255, 255, 0.3), 20px 0 0 2px rgba(255, 255, 255, 0.3), 40px 0 0 2px rgba(255, 255, 255, 0.3);
+    content: '';
+    display: block;
+    height: 7px;
+    left: 8px;
+    position: absolute;
+    top: -18px;
+    width: 7px;
+}
+.windows #summary .terminal:before {
```

Comment:
this should use `::before` to match the modern CSS spec and not the legacy alias (our usage of custom properties makes the profiler UI incompatible with browsers that are so old that they don't even support `::before`).

`::before` is a pseudo-element not a pseudo-class. And as of CSS 3, pseudo-elements use `::` as syntax (`before` and `after` were the only pseudo-elements defined in CSS 2 and so the only ones being supported through a legacy alias using the pseudo-class syntax)

Reply:
Updated. Thanks for the explanation.